### PR TITLE
Added functionality in display_best_score function

### LINKED ISSUE #23: Add code for display_best_score function
Add code for display_best_score function

### PR DESCRIPTION
**Pull request by AnuBanga**

### CODE REVIEW

File: src/autopredictor/bestscore.py [hunk 1]
```diff
@@ -13,6 +16,44 @@ def display_best_score(X, scoring_metric):
...
+    if X.empty:
+        raise TypeError("DataFrame is empty.")
+    
+    if not isinstance(scoring_metric, str):
+        raise ValueError("scoring_metric must be a string.")
+
+    if scoring_metric not in X.columns:
+        available_metrics = X.columns.tolist()
+        available_metrics_string = ", ".join(available_metrics)
+        raise ValueError (f"Invalid Scoring metric '{scoring_metric}'.The specified metric is not in the list of available metrics. Available metrics: {available_metrics_string}.")
```
Comment (zgarciaj) [inline src/autopredictor/bestscore.py]:
```suggestion
        raise ValueError (f"Invalid Scoring metric '{scoring_metric}'. The specified metric is not in the list of available metrics. Available metrics: {available_metrics_string}.")
```
Grammar (if changed here, it must be changed in the test file)

**Review by zgarciaj — CHANGES_REQUESTED** [review]
Great job, @AnuBanga ! The display_best_score function is well-documented and is efficient at providing clear error messages. It's good at handling different edge cases that ensure reliability and usability in diverse scenarios.

I made a minor grammar suggestion. If this grammar suggestion is accepted, please ensure the same grammar suggestion is applied to the test file of this function. 

Other than that, I believe this file is good for merger!

**Review by zgarciaj — APPROVED** [review]
